src/cli.c

```diff
@@ -97,6 +109,7 @@ static void print_help(void) {
            "  -v, --version                 print tjs version\n"
            "  -h, --help                    list options\n"
            "  -e, --eval EXPR               evaluate EXPR\n"
+           "  -l, --load FILENAME           module to preload (option can be repeated)\n"
```

Comment:
Also, I wonder if we should support passing a single option, with comma separated items. WDYT?

Reply:
I don't think that is a good idea because filenames are allowed to contain both comma and whitespace characters making it impossible to disambiguate between _exotic_ filenames and list entry separators. Apart from that, it is pretty non-standard and I don't see the particular benefit over just going with repeated flags.

Reply:
👍